.eslintrc

```
@@ -0,0 +1,17 @@
+{
+  "root": true,
+  "extends": [
+    "airbnb-base",
+    "plugin:prettier/recommended",
+    "plugin:node/recommended"
+  ],
+  "rules": {
+    "prettier/prettier": "error",
```

```suggestion
    "prettier/prettier": ["error", {
      "tabWidth": 4,
      "singleQuote": true,
      "trailingComma": "es6",
      "endOfLines": "LF"
    }],
```

prettier の設定内容がそこまで多くないので
prettier.yml 作らずに ↑ に書くのはどうでしょ
あと .eslintrc は *.json か *.js にできればしたいです
YAML でもいいけど、JSON か JS の方が他の人も書きやすいかなと

---

ref 1:
- https://github.com/airbnb/ts-migrate/blob/master/.eslintrc.js
- https://github.com/codeforosaka/covid19/blob/development/.eslintrc

---

ref 2: https://eslint.org/docs/user-guide/configuring

---

prettierrcはこれくらいの量だと直接記入でいいですね了解っす
.eslintrcは.eslintrc.jsonに書き換えときます

---

@75asa 
```
"tabWidth": 4,
"singleQuote": true,
```
はairbnbの設定と競合するようで、適用されないので消しておきます